tensorflow/lite/micro/tools/make/targets/bluepill_makefile.inc

```diff
@@ -1,64 +1,60 @@
-# Settings for Blue Pill platforms.
-ifeq ($(TARGET), bluepill)
+export PATH := $(MAKEFILE_DIR)/downloads/gcc_embedded/bin/:$(PATH)
```

Comment:
The changes to this file are removing the unnecessary ifeq TARGET at the top of the file (since only this makefile is included) and removing the leading whitespace on all the other lines.

The diff is not helping here.